<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Solution_show</class>
 <widget class="QDialog" name="Solution_show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>换乘路线</string>
  </property>
  <widget class="QPushButton" name="button_compare">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>570</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>comapre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>570</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_OK">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>570</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>631</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser_Solution"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
